ui/App/UserProfileScreen.svelte

```diff
@@ -6,35 +6,61 @@
  LICENSE file.
 -->
 <script lang="typescript">
+  import type { Project } from "ui/src/project";
+  import { fetchProjects, projects as store } from "ui/src/userProfile";
```

Comment:
Renaming `projects` to the generic `store` makes it harder to understand what is going on when one reads `store`.

```suggestion
  import { fetchProjects, projects } from "ui/src/userProfile";
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2398/commits/485ac38bc0f582ae01cf121375c22e1245297692